java/demo/src/com/google/phonenumbers/PhoneNumberParserServlet.java

```diff
@@ -161,9 +194,31 @@ private void appendLine(String title, String data, StringBuilder output) {
   }
 
   /**
+   * Returns a stable URL pointing to the result page for the given input.
+   */
+  private String getPermaLinkURL(
+      String phoneNumber, String defaultCountry, String languageCode, String regionCode)
+      throws UnsupportedEncodingException {
+    StringBuilder permaLink = new StringBuilder();
+    permaLink.append("http://libphonenumber.appspot.com/phonenumberparser");
```

Comment:
I can't make it relative as it has to work from github. (That is, unless we don't link back from the github issue.)